src/encoded/static/components/patholgoyReprotTable.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { Panel, PanelBody, PanelHeading } from '../libs/bootstrap/panel';
+import Status from './status';
+import CollapsiblePanel from './collapsiblePanel';
+
+
+class PathologyReportTable extends React.Component {
+    constructor(props) {
+        super(props);
+        this.filteredData = [];
+        this.tableId = [];
+    }
+
+    filterData() {
+        this.filteredData = this.props.data;
+       
+
+        for (let i = 0; i < this.filteredData.length; i++) {
+            let tableTitle = this.props.tableTitle + "Tumor information" + " " + (i + 1);
+            let tableIdi =
+                (< div >
+
+                        <PanelBody>
+                            <dl className="key-value">
+                                <div data-test="status">
+                                    <dt>Status</dt>
+                                    <dd><Status item={this.filteredData[i].status} inline /></dd>
+                                </div>
+
+                                <div data-test="date">
+                                    <dt>Pathology Report Date</dt>
+                                    <dd>{this.filteredData[i].date}</dd>
+                                </div>
+                                <div data-test="laterality">
+                                    <dt>Laterality</dt>
+                                    <dd>{this.filteredData[i].laterality}</dd>
+                                </div>
+                                <div data-test="tumor_size">
+                                    <dt>Tumor_size</dt>
+                                    <dd>{this.filteredData[i].tumor_size}{this.filteredData[i].tumor_size_units}</dd>
+                                </div>
+
+                                <div data-test="focality">
+                                    <dt>Focality</dt>
+                                    <dd>{this.filteredData[i].focality}</dd>
+                                </div>
+                                <div data-test="histology">
+                                    <dt>Histology</dt>
+                                    <dd>{this.filteredData[i].histology}</dd>
+                                </div>
+                                <div data-test="sarcomatoid">
+                                    <dt>Sarcomatoid</dt>
+                                    <dd>{this.filteredData[i].sarcomatoid}</dd>
+                                </div>
+                                <div data-test="sarcomatoid_percentage">
```

Comment:
Hide the field sarcomatoid_percentage if there is no value for this field

Reply:
We can filter all the show fields after we decide which fields need to be shown. Agree to fix this.